tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+const production = !process.env.ROLLUP_WATCH;
+
+module.exports = {
+  future: {},
+  purge: {
+    content: ["./src/**/*.svelte", "./src/**/*.html"],
+    enabled: production, // disable purge in dev
+    options: {
+      whitelist: [/svelte-/],
```

Comment:
In [version 3](https://github.com/FullHuman/purgecss/blob/master/CHANGELOG.md) of purgeCSS, `whitelist` is now called `safelist`.

Reply:
Extra info: PurgeCSS v3.x is now included in TailwindCSS v1.9 and above. So this will need to be changed for this to work.